File: client/common/model/feedback/process.ts

export class Process {
  constructor(
    public id: number,
    public feedbackId: number,
    public userId: number,
    public result: string,
  ) {
    this.id = id
    this.feedbackId = feedbackId
    this.userId = userId
    this.result = result
  }
}

File: server/internal/feedback/models/process.go
package models

import (
	"server/common/middleware/database"
)

type Process struct {
	database.BaseEntity
	FeedbackID uint64 `json:"feebackId"`
	UserID     uint64 `json:"userId"`
	Result     string `json:"result"`
}

func (r *Process) TableName() string {
	return "feedback_process"
}
